ml-proto/host/builtins.ml

```diff
@@ -1,9 +1,50 @@
 open Source
+open Eval
```

Comment:
From a readthrough, the only use of the Eval module in this file is the Eval.memory_for_module. Can this open be removed?